taar/adapters/dynamo.py

```diff
@@ -0,0 +1,53 @@
+import boto3
+import json
+
+
```

Comment:
Let's add a `logger` (as done in the other modules) and log things where needed (e.g. errors).